_includes/head.html

```diff
@@ -12,9 +12,12 @@
   <script type="text/javascript" charset="utf-8" src="{{ "/js/plugins.js" | prepend: site.baseurl }}"></script>
   <script type="text/javascript" charset="utf-8" src="{{ "/js/main.js" | prepend: site.baseurl }}"></script>
   <script type="text/javascript" charset="utf-8" src="{{ "/js/vendor/modernizr-2.6.2.min.js" | prepend: site.baseurl }}"></script>
+  <script type="text/javascript" charset="utf-8" src="{{ "/js/vendor/jsondoc-forwarder-0.3.0.js" | prepend: site.baseurl }}"></script>
```

Comment:
Do we really want to load the redirect scripts on every page? Why load the entire existing HTML file and then redirect? Why not delete all the HTML content, have the site load the `404.html` file, and redirect from there?

Reply:
I want to (at least for now) preserve the TOC pages on the existing gh-pages docs. There isn't anything like that currently for the devsite.

Reply:
Can't we keep those TOC pages, change the links on them, delete all the other files, and have `404.html` handle all the other redirects?

Reply:
Done. Gonna wait on resolving merge conflicts until https://github.com/googleapis/google-cloud-ruby/pull/3684 is merged, otherwise they'll keep happening